arch/arm/src/s32k3xx/s32k3xx_edma.c

```diff
@@ -614,6 +620,22 @@ static inline void s32k3xx_tcd_configure(struct s32k3xx_edmatcd_s *tcd,
                                   EDMA_TCD_CSR_INTHALF : 0;
   tcd->dlastsga = config->flags & EDMA_CONFIG_LOOPDEST ? -config->iter : 0;
 
+#ifdef CONFIG_S32K3XX_DTCM_HEAP
+  /* Remap address to backdoor address for eDMA */
+
+  if (tcd->saddr >= (uint32_t)DTCM_BASE_ADDR
+      && tcd->saddr < (uint32_t)DTCM_END_ADDR)
```

Comment:
I'm not sure which of this or
```suggestion
  if (tcd->saddr >= (uint32_t)DTCM_BASE_ADDR &&
      tcd->saddr < (uint32_t)DTCM_END_ADDR)
```
is recommended by code style. Let me check